monitoring/grafana/dashboards/ceph-cluster.json

```
@@ -402,52 +403,52 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "ceph_pg_total",
+          "expr": "ceph_pool_metadata * on(pool_id,instance) group_left() (ceph_pg_total)",
```

```suggestion
          "expr": "sum(ceph_pg_total)",
```

Lets just aggregate the per-pool metrics for the cluster dashboard.

---

Done, also reverted legend format.